low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -190,12 +190,21 @@ def __init__(self, args):
         super().__init__(args)
 
     def validate(self):
-        # Common validator classes to check Cortx/system wide validator
-        pass
+        if self.args and len(self.args) > 1:
+            i=0
+            while i < len(self.args):
+                try:
+                    if self.args[i] == "sanity":
+                        break
+                    elif self.args[i] == "alerts":
+                        break                   
+                except (IndexError, ValueError):
+                    raise SetupError(errno.EINVAL, "Invalid plan for %s"% self.name)
+                i+=1
 
     def process(self):
-        # TODO: Import relevant python script here for further execution.
-        pass
+        from cortx.sspl.lowlevel.files.opt.seagate.sspl.setup.sspl_test import SSPLTestCmd
+        SSPLTestCmd(self.args).process()
```

Comment:
Command would be `sspl_setup test sanity`. 
Here self.args holds two elements in it.  ['test', 'sanity']
Please make sure sspl_test is taking correct one from this as a plan.

Reply:
Done the changes.
JFYI, self.args will hold only ['sanity']. As the test arg is used by the parser 
to determine the type of class required to call.

Reply:
Okay, I got it.